### PR TITLE
adding the l1 loss in paddle.nn.functional.loss

### LINKED ISSUE #19271: l1_loss

### PR DESCRIPTION
**Pull request by ZakariaR1ad**
Close #19271

### CODE REVIEW

**Review by Daniel4078 — APPROVED** [review]
Looks good to me!
